src/components/MiddleChatWindow.js

```diff
@@ -29,7 +31,14 @@ function MiddleChatWindow({ currentUser }) {
 
   return (
     <>
-      <ul className='list-container d-flex list-unstyled'>
+      <ul
+        className='list-container d-flex list-unstyled''
```

Comment:
```suggestion
        className='list-container d-flex list-unstyled'
```

I'm getting an error here.
![image](https://user-images.githubusercontent.com/8095975/187987245-a0e92d97-fbd8-41a1-8e04-efc04a5aab9d.png)